addplayer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=halloffame', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare('SELECT `name`, `age`,`nationality`,`goal`,`image` FROM `Players`;');
```

Comment:
Why are you querying the database on this page? Doesn't look like you need lines 5-7 if you're not displaying the data on this page at all